Sluggo/Models/Codables/Team.swift

```diff
@@ -8,13 +8,21 @@
 import Foundation
 
 
-struct TeamRecord: Codable {
+struct TeamRecord: Codable, Equatable {
     var id: Int
     var name: String
     var object_uuid: UUID
     var ticket_head: Int
     var created: Date
     var activated: Date?
     var deactivated: Date?
-   
+    
+    static func == (lhs: TeamRecord, rhs: TeamRecord) -> Bool {
+        return lhs.id == rhs.id &&
+            lhs.object_uuid == rhs.object_uuid &&
```

Comment:
if the UUID is indeed unique, could we not save time by simply comparing the UUID? or is there a specific intended purpose of this equality check?

Reply:
there is no specific purpose I'm just dumb